tests/integration/test_reverse_dns_resolution_concurrent/configs/config.xml

```diff
@@ -0,0 +1,3 @@
+<clickhouse>
+    <max_concurrent_queries>250</max_concurrent_queries>
```

Comment:
Should we also disable DNS cache?

Reply:
I guess it does make more sense for this test, I didn't disable it because the crash was happening during crash updates. But will enable it.

Reply:
Then we can change the update period setting, so DNS cache will be updated each second

Reply:
I guess that's better indeed, what's the setting for that?

Reply:
dns_cache_update_period